2021q3/boot-performance.adoc

```diff
@@ -0,0 +1,30 @@
+== Boot Performance Improvements
+
+Links
+link:https://wiki.freebsd.org/BootTime[Wiki page]
+link:https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html[OS boot time comparison]
+
+Contact: Colin Percival <cperciva@FreeBSD.org>
+
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is using an EC2 c5.xlarge instance as a reference
+platform and is measuring the time between when the virtual machine enters the
+EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017 and led to a conference presentation, "Profiling
```

Comment:
s/2017 and led/2017, leading to/ (flow: avoid "and" repetition)

Reply:
Thanks!